### DISCUSSION TITLE
how would I put this in github pages (I'm new)

### DISCUSSION THREAD

**Discussion by 1jammer1** [copy/v86#826] [Q&A]
I know this is like another discussion but that did not answer my question so how can I do it and can I do it please help Im new to this

**Comment by Darin755** [comment]
You should have a look at the examples and the starter.js.

If you looking for a simple solution you could just embed v86 in a iframe.

Another alternative is my current project. My primary goal is to make v86 easier to work with for embedding Linux on web pages. It is still a work in progress but it does have an embed option https://github.com/darin755/browser-linux
